readme.md

```diff
@@ -707,6 +707,7 @@
 - [Executable Packing](https://github.com/dhondta/awesome-executable-packing#readme) - Packing and unpacking executable formats.
 - [Malware Persistence](https://github.com/Karneades/awesome-malware-persistence#readme) - Techniques that adversaries use to keep access to systems across restarts.
 - [EVM Security](https://github.com/kareniel/awesome-evm-security#readme) - Understanding the Ethereum Virtual Machine security ecosystem.
+- [Password Cracking](https://github.com/n0kovo/awesome-password-cracking#readme) - Tools, research, papers and other projects related to password cracking and password security.
```

Comment:
The description should describe the topic not the list itself.

Reply:
I've added a further explanation of the topic:
https://github.com/n0kovo/awesome-password-cracking/commit/41a44b8d4890a9ba7a9a2d35f5d4135a7b9b4456

Reply:
Fixed